sos/plugins/networking.py

```diff
@@ -154,6 +155,54 @@ def setup(self):
             pred=SoSPredicate(self, kmods=['ip6table_filter'])
         )
 
+        # Get devlink output
+        self.add_cmd_output([
+                    "devlink dev",
+                    "devlink dev param show",
+                    "devlink dev info",
+                    "devlink port",
+                    "devlink sb show",
+                    "devlink sb pool show",
+                    "devlink sb port pool show",
+                    "devlink sb tc bind show",
+                    "devlink -s -v trap show"
+        ])
+
+        # Collect information about devlink devices and dpipe tables
+        devlink_dev = self.exec_cmd("devlink dev")
```

Comment:
Should we save `devlink dev` output? Is it ever useful on its own? If so, convert `exec_cmd` to `collect_cmd_output` here.

Reply:
And now I see we're calling it separately above. So yeah, here we can convert to `collect_cmd_output` and then remove the `devlink dev` entry from line 160.